docs/quotas.md

```diff
@@ -0,0 +1,13 @@
+# Resource Quota Tiers
+
+When requesting a namespace, you can choose from 1 of 3 tier options for your resource quota:
+
+|        | CPU Cores | Memory | Storage |
+|--------|-----------|--------|---------|
+| Small  |     1     |   4Gi  |   20Gi  |
+| Medium |     2     |   8Gi  |   40Gi  |
+| Large  |     4     |  16Gi  |   80Gi  |
+
+# Requesting custom quotas
+
+Please make an issue in this repo if your project has more custom requirements.
```

Comment:
I'd change it to `Please make an issue in this repo if your project has requirements that are not met by the default quotas.`